erdpy_network/transaction_logs.py

```diff
@@ -20,3 +20,14 @@ def from_http_response(logs: Dict[str, Any]) -> 'TransactionLogs':
         result.events = [TransactionEvent.from_http_response(item) for item in events]
 
         return result
+    
+    def find_first_or_none_event(self, identifier: str, predicate: Callable[[TransactionEvent], bool] = None) -> TransactionEvent:
```

Comment:
It says `...or_none`, but the result does not include None (typing annotation etc.).

Reply:
Done.